scripts/automation/wait_and_collect_coderabbit.py

```
@@ -0,0 +1,31 @@
+import sys
+from pathlib import Path
+
+branch_file = Path(sys.argv[1] if len(sys.argv) > 1 else ".tutti/state/auto/branch.json")
+wait_mins = sys.argv[2] if len(sys.argv) > 2 else "45"
+status_file = sys.argv[3] if len(sys.argv) > 3 else ".tutti/state/auto/coderabbit_status.json"
+feedback_file = sys.argv[4] if len(sys.argv) > 4 else ".tutti/state/auto/coderabbit-feedback.md"
+
+with branch_file.open("r", encoding="utf-8") as f:
+    branch = json.load(f)["branch"]
+
+out = subprocess.check_output(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)